### PR TITLE
refactor: spell_insert

### PR DESCRIPTION
**Pull request by JimmyDaddy**
* refactor spellBook insert commands, `upsert` and `bulkInsert` base on `insert`
* fix `find` API in Sequelize mode
* `Bone.upsert`

### CODE REVIEW

**Comment by JimmyDaddy** [comment]
ci: https://github.com/JimmyDaddy/leoric/runs/2897478787

File: lib/bone.js [hunk 1]
```diff
@@ -1339,6 +1388,7 @@ class Bone {
       if (attributes.hasOwnProperty(snakeCase(name))) {
         timestamps[name] = snakeCase(name);
       }
+      if (attributes.hasOwnProperty(LEGACY_TIMESTAMP_MAP[name])) timestamps[name] = LEGACY_TIMESTAMP_MAP[name];
```
Comment (JimmyDaddy) [inline lib/bone.js:1391]: `Bone.init` isn't an async function , cannot  call `querySchemaInfo` in it, so take a `LEGACY_TIMESTAMP_MAP ` patch here too.

File: lib/drivers/abstract/spellbook.js [hunk 2]
```diff
@@ -756,6 +840,7 @@ module.exports = {
 
     switch (spell.command) {
       case 'insert':
+      case 'bulkInsert':
```
Comment (JimmyDaddy) [inline lib/drivers/abstract/spellbook.js:843]: Whether merges `bulkInsert` and `insert` actions  are unified as `insert`?

**Review by cyjake — APPROVED** [review]
+1

**Pull request merged**